### PR TITLE
Yolo project join Slingshot

### PR DESCRIPTION
**Pull request by yuwenhui**
Yolo is a decentralized images dataset to develop ML and AI models based on Filecoin and IPFS. We are planning to store [Open Images Dataset](https://storage.googleapis.com/openimages/web/index.html) to Filecoin and IPFS network by Powergate, and continue to enrich and improve it to be a decentralized dataset for computer vision research.

### CODE REVIEW

**Comment by dineshshenoy** [comment]
Thanks for submitting an application.  Our team will validate for completeness and then assign reviewers from the competitions review panel.

File: participants/yolo.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+We are using a curated dataset.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We are planning to store at least 5TB data to the Filecoin network, maybe more than 10TB if everything goes well.
+
+## (7) How are you structuring the data?
+The images dataset are unstructured data with JPG or PNG format. However, we manage these dataset and our application data by database like MySQL.
+
+## (8) What pre-processing are you doing before ingesting the data?
+It is currently planned to package a series of images which have same type and category, and then store them to the Filecoin network by sending storage deals to miners. The size of storage deals will be 3GB to 5GB according to the network bandwidth when sending to miners.
```
Comment (dineshshenoy) [inline participants/yolo.md:31]: I believe there may currently be a technical limitation of 4GB for deals in Filecoin.  You're welcome to do larger deals subject to technical feasibility.

File: participants/yolo.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Every image which has packaged will be saved as a record in our database also. There is a `storageDealId` field to record that each image is contained in which deal. We get the deal status by subscribing Powergate's JSON RPC API. After one storage deal has successfully be sealed on the Filecoin sector, we will send a retrieve deal, and then store all the images in the deal to IPFS network as cache. The Yolo application will fetch images and expose them to users from IPFS network.
+
+There is some uncertainty about the API of Powergate, we have not yet completed the integration of Powergate's api, perhaps something will change our plan.
+
+## (9)  What tech stack will you use to store the data?
+We are planning to use Powergate and lotus.
```
Comment (dineshshenoy) [inline participants/yolo.md:38]: Since Powergate has a lotus node built-in, do you need to use both?

File: participants/yolo.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## (10) How will you retrieve the data?
+I think our application only need to retrieve data from Filecoin once. After the storage deal has successfully sealed by miner, we will send a retrievel deal, and then store all the data in the deal to IPFS. IPFS is a intermediate caching layer for us to retrieve the necessary data and expose it to users.
+
```
Comment (dineshshenoy) [inline participants/yolo.md:42]: Will retrieval be on a per image basis or on a set of images?

File: participants/yolo.md (hunk 1 above)
Comment (yuwenhui) [reply participants/yolo.md:31]: Okey, we will try large size deals, get in touch.

File: participants/yolo.md (hunk 2 above)
Comment (yuwenhui) [reply participants/yolo.md:38]: Only Powergate's API, i found that Powergate is friendly for developers when i implement some features currently.

File: participants/yolo.md (hunk 3 above)
Comment (yuwenhui) [reply participants/yolo.md:42]: Will retrieval be on a set of images. We are planning to package a set of images for about 3GB to 5GB, and store them as storage deals to Filecoin, and then retrieve these deals and un-package them as single image, each image will be stored to IPFS as cache for expose it to users. I don't clearly does Powergate's API support this case, please let us try.

**Comment by dineshshenoy** [comment]
Congratulations. Your application has been approved and merged as an official participant for the Slingshot competition!
The following resources may be helpful as you begin:

[Competition Home](https://slingshot.filecoin.io/)
[Competition Rules](https://slingshot.filecoin.io/rules)
[FAQs](https://slingshot.filecoin.io/faq)
[Events calendar](https://slingshot.filecoin.io/events/) 
Office Hours
Master Classes
[Filecoin Docs](https://docs.filecoin.io) 
[HackFS videos](https://hack.ethglobal.co/showcase/myel-receOUiPSd63148qZ)
[Textile Powergate](https://docs.textile.io/powergate/)

Feel free to reach out in the slack community channels:
#fil-testnet-announce – For major announcements about the testnet and major software releases
#slingshot-announcements – For major announcements about the Slingshot program
#slingshot – For debugging help and community conversations throughout the course of the competition

**Pull request merged**